index.xml

```diff
@@ -8607,6 +8725,10 @@ context.setColorEnforcedRange(-1, 255, 256);</x:codeblock>
                   <dd>The <a class='dfnref' href='#dfn-exposure-set'>exposure set</a> of the interface member
                   is the <a class='dfnref' href='#dfn-exposure-set'>exposure set</a> of the interface or
                   partial interface the member is declared on.</dd>
+                  <dt>namespace member</dt>
+                  <dd>The <a class='dfnref' href='#dfn-exposure-set'>exposure set</a> of the
+                  namespace member is the <a class='dfnref' href='#dfn-exposure-set'>exposure
+                  set</a> of the namespace the member is declared on.</dd>
```

Comment:
Need to clone the "The [Exposed] extended attribute MUST NOT be specified on both an interface member and a partial interface definition the interface member is declared on." thing for namespaces, right?